safe-global/web-core
# build: migrate to pnpm

## What it solves

Migration to `pnpm`

## How this PR fixes it

`pnpm` has been installed and configured for usage. A temporary script has been added that prevents usage of any other package managers for installing dependencies.

`yarn.lock` has also been removed.

## How to test it

Run `pnpm install` and observe the population of `node_modules` (especially `node_modules/.pnpm`. All other commands run via a package manager should also run, e.g. `lint` and `test`.
